### DISCUSSION TITLE
Where do I start?

### DISCUSSION THREAD

**Discussion by Toastandjam0** [filebrowser/filebrowser#2631] [Q&A]
Hi,
I have managed to load Filebrowser on my Synology NAS, using Docker, and have logged in , expecting to view the various files on my NAS.
But there is nothing there - just a 'It's lonely here' message.
How does Filebrowser work?
I would appreciate some kind advice on what I may have missed or made  a mistake with. 
Could it be the filepath that is listed under 'Environment' ?
I'd be grateful for any assistance.
![filebrowser](https://github.com/filebrowser/filebrowser/assets/141869379/326ab058-ec0e-4ca8-bd87-7549239e7f35)

I am beginning to feel out of my depth.

Stephen

**Comment by masterwishx** [comment]
Did you linked your volume to /srv ? Then add to /srv/somesharename

![image](https://github.com/filebrowser/filebrowser/assets/28630321/39777e2c-0aef-4102-be71-fe8d6c59f19f)

**Comment by Toastandjam0** [comment]
Hi,
Thanks for your assistance, I really appreciate the effort to send a reply.

The trouble is, I’m an old guy, trying to get his Family Website to run smoothly, and generally it does, but when bits start to fall apart, I get sucked into suggestions by posters online, and in this case have ended up going down the File Browser path totally blind, not really knowing what I’m doing!

The reason I started with FileBrowser, was to access the PHP.info file in the NAS and make an edit. I’ve since established that such a facility existed on the Synology Web Station menu, which has half fixed my problem, at least made it bearable.

I think GitHub is a bit too deepend for me, but thanks again for your suggestions.

Stephen

Trees for Cities<https://www.treesforcities.org/our-work>
Trees for Life<https://treesforlife.org.uk/plant-trees/grove/7635/>

**Comment by leolee82** [comment]
map your local folder to /srv
